src/frontend/src/components/details/Details.tsx

```diff
@@ -198,7 +198,13 @@ function TableStringValue(props: FieldProps) {
   }
 
   return (
-    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        wordBreak: 'break-word'
```

Comment:
I would add an alignItems: flex-start too, so that the icon, field name starts at the top of the container.